### PR TITLE
8254982: (tz) Upgrade time-zone data to tzdata2020c

### PR DESCRIPTION
**Pull request by kiranoracle**
Hi Guys,

Please review the integration of tzdata2020c to JDK. 

Details regarding the change can be viewed at - https://mm.icann.org/pipermail/tz-announce/2020-October/000060.html
Bug: https://bugs.openjdk.java.net/browse/JDK-8254982

Along with it, there is a test fix for https://bugs.openjdk.java.net/browse/JDK-8254865, With tzdata2020b, the test fails for the mentioned zones expecting "PST" but JDK has Zone names for "MST" for JRE locale provider. Since the purpose of the test is to get any GMT-08:00 time zone, I have excluded those zones from the test.

Please let me know if the changes are good to push.

Thanks,
Kiran

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8254982](https://bugs.openjdk.java.net/browse/JDK-8254982): (tz) Upgrade time-zone data to tzdata2020c

### Reviewers
 * [Erik Joelsson](https://openjdk.java.net/census#erikj) (@erikj79 - **Reviewer**) ⚠️ Review applies to 7bd515373aa2193e12f444b2ea05a024761d9344
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/747/head:pull/747`
`$ git checkout pull/747`

### CODE REVIEW

**Review by erikj79 — APPROVED** [review]
Build change looks ok.

File: test/jdk/java/util/Formatter/BasicDateTime.java [hunk 1]
```diff
@@ -1687,6 +1687,14 @@ public static void test() {
 
         // Get the supported ids for GMT-08:00 (Pacific Standard Time)
         String[] ids = TimeZone.getAvailableIDs(-8 * 60 * 60 * 1000);
+
+        final List<String> list =  new ArrayList<String>();
+        Collections.addAll(list, ids);
+        list.remove("America/Dawson");
+        list.remove("America/WhiteHorse");
+        list.remove("Canada/Yukon");
```
Comment (naotoj) [inline test/jdk/java/util/Formatter/BasicDateTime.java:1695]: I'd explicitly mention why these time zones are removed from the test.

Comment (kiranoracle) [reply test/jdk/java/util/Formatter/BasicDateTime.java:1698]: Thanks, Naoto, I have added the information.

**Review by naotoj — APPROVED** [review]
Looks good. Thank you for the changes.

**Pull request closed without merging**